jolly-jellyfish/src/static/themes/win95.css

```diff
@@ -28,6 +29,20 @@ body {
   margin: 30px auto 60px auto;
 }
 
+img {
+  width: 100%;
+  height: 100px;
```

Comment:
Should we really be using pixel measurements for something?

I'm not sure if it appears anywhere else though, so maybe we already are and I missed it...

Reply:
There are quite a few things using pixels, I take your point for this one though, maybe I should change it to 2em or something.